compiler/grainlsp/grainlsp.re

```diff
@@ -0,0 +1,48 @@
+//let _ = Lspserver.run();
+
+open Printf;
+open Cmdliner;
+
+let lsp_wrapper = () => {
+  let _ = Lspserver.run();
+
+  `Ok();
+};
+
+let lsp = opts =>
+  try(lsp_wrapper()) {
+  | exn =>
+    Format.eprintf("@[%s@]@.", Printexc.to_string(exn));
+    exit(2);
+  };
```

Comment:
Is it possible to `try(Lspserver.run()) { ... }` or do we need the `lsp_wrapper` to catch the exn here or something?

Reply:
Cleaned up to pass proper values back instead of throwing the exception